cpu/nrf52/periph/Kconfig

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) 2021 HAW Hamburg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+
+config MODULE_PERIPH_UART_NONBLOCKING
+    depends on HAS_PERIPH_UART_NONBLOCKING
+    depends on MODULE_PERIPH_UART
+    select MODULE_TSRB
+
+config MODULE_PERIPH_SPI
+    depends on HAS_PERIPH_SPI
+    select MODULE_PERIPH_GPIO_IRQ if CPU_MODEL_NRF52832XXAA
```

Comment:
```suggestion
    select MODULE_PERIPH_GPIO_IRQ if CPU_MODEL_NRF52832XXAA && HAS_PERIPH_GPIO_IRQ
```

Reply:
Can you explain why this is better? If I know that the CPU_MODEL has the periph?

Reply:
I often find explicit better than implicit. Maybe if this is moved to the `CPU_MODEL_NRF52832XXAA` declaration this is clearer and we can avoid checking for `HAS_PERIPH_GPIO_IRQ` as it is `select`ed in the same place, so it's more obvious.

Just trying to make things as clearer and more obvious.

Reply:
addressed